### ISSUE TITLE
CRON job error on e-mailadres starting with a single quote

### ISSUE THREAD

**Issue by wderuijter** [mautic/mautic#1495]
## Description
The CRON job mautic:campaigns:trigger --env=prod -f gave this error:
```
  [RuntimeException]
  The mcrypt extension is not enabled on this server.
```
But the mcrypt extension is installed on my server. I confirmed this with phpinfo();

I had a lead with an emailadres starting with a single quote ('), probably due to a typo in a form somewhere.
After fixing his email address the error was not there anymore. This might be related.

**Comment by escopecz** [comment]
I don't think the error about mcrypt extension not being enabled and a single quote in an email address can be related. Or can you provide steps to replicate?

**Comment by wderuijter** [comment]
This is the error log:
`[2016-03-02 08:44:01] mautic.ERROR: RuntimeException: The mcrypt extension is not enabled on this server. (uncaught exception) at /var/www/vhosts/roc-nijmegen.nl/httpdocs/marketing/app/bundles/CoreBundle/Helper/EncryptionHelper.php line 31 while running console command `mautic:campaigns:trigger` [] []`

I have to check on a test installation if I can replicate it.

**Comment by escopecz** [comment]
> The mcrypt extension is not enabled on this server

Install and enable the mcrypt PHP extension and then restart the server to apply the change. It's not a Mautic imagination that the mcrypt is missing. It's PHP error. Maybe contact your server provider and consult it with him.

**Comment by wderuijter** [comment]
This install is running for several months already with mcrypt installed building thousands of mautic leads without this mcrypt error. 
phpinfo() shows mcrypt as enabled.
```
if(function_exists("mcrypt_encrypt")) {
    echo "mcrypt is loaded!";
}
if (extension_loaded('mcrypt')){
	echo "loaded";
}
```
This returns "loaded loaded"

Weird thing is this error only occurred a couple of times and only this morning after forcing the cron job. After removing the incorrect email address (I stumbled upon by accident) the error seems to be resolved and the cron jobs run without errors.

It might have been caused by another fluke.
